### PR TITLE
feat(articles): implements articles index page

### PR DESCRIPTION
**Pull request by dzucconi**
Sitting under /articles2 for now.

## TODO

- [ ] Add `featured` and `published` arguments to `articlesConnection` field.
- [ ] Migrate RSS feed code

### CODE REVIEW

File: src/v2/Apps/Articles/ArticlesApp.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { Link } from "react-head"
+import { Spacer, Text } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { MetaTags } from "v2/Components/MetaTags"
+import { ArticlesIndexArticlesPaginationContainer } from "./Components/ArticlesIndexArticles"
+import { getENV } from "v2/Utils/getENV"
+
+const ArticlesApp = ({ viewer }) => {
```
Comment (damassi) [inline src/v2/Apps/Articles/ArticlesApp.tsx:8]: Should this have a type?

Comment (dzucconi) [reply src/v2/Apps/Articles/ArticlesApp.tsx:8]: Ah, woops thank you

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Routes added to `routes.tsx` should have a corresponding cypress.js smoke test. See the `cypress/integration` folder for examples.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against ce4e30744fc4d6eaf4123caabcf7cc8a07ba2bfe
</p>

**Review by damassi — DISMISSED** [review]
Sweet 👍

**Pull request merged**